coverage/.last_run.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
You need to put the coverage folder in the gitignore.
To remove this from version control you need to remove the git cache

https://devconnected.com/how-to-clear-git-cache/